### PR TITLE
Update 0042-LIQF-setting_fees_and_rewarding_lps.md

### PR DESCRIPTION
**Pull request by vega-paul**

### CODE REVIEW

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 1]
```diff
@@ -271,8 +271,19 @@ When the time defined by `market.liquidity.providers.fee.distributionTimeStep` e
 - If a market has `market.liquidity.providers.fee.distributionTimeStep` set to more than `0` and such market settles then the fees are distributed as part of the settlement process, see [market lifecycle](./0043-MKTL-market_lifecycle.md). Any settled market has zero balances in all the LP fee accounts. (<a name="0042-LIQF-014" href="#0042-LIQF-014">0042-LIQF-014</a>)
 - All liquidity providers with `average fraction of liquidity provided by committed LP > 0` in the market receive a greater than zero amount of liquidity fee. The only exception is if a non-zero amount is rounded to zero due to integer representation. (<a name="0042-LIQF-015" href="#0042-LIQF-015">0042-LIQF-015</a>)
 
-### API
-
+### LP JOING AND LEAVING MARKETS 
+- An LP joining a market that is below the target stake with a higher fee bid than the current fee: their fee is used (<a name="0042-LIQF-019" href="#0042-LIQF-019">0042-LIQF-019</a>)
```
Comment (davidsiska-vega) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md:275]:
```suggestion
- An LP joining a market that is below the target stake with stake that will take the market over the target with a higher fee bid than the current fee: their fee is used (<a name="0042-LIQF-019" href="#0042-LIQF-019">0042-LIQF-019</a>)
```

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md [hunk 2]
```diff
@@ -271,8 +271,19 @@ When the time defined by `market.liquidity.providers.fee.distributionTimeStep` e
...
+- An LP joining a market that is above the target stake with a commitment large enough to push one of two higher bids above the target stake, and a higher fee bid than the current fee: the fee doesn't change (<a name="0042-LIQF-024" href="#0042-LIQF-024">0042-LIQF-024</a>)
+- An LP leaves a market that is above target stake when their fee bid is currently being used: fee changes to fee bid by the LP who takes their place in the bidding order (<a name="0042-LIQF-025" href="#0042-LIQF-025">0042-LIQF-025</a>)
+- An LP leaves a market that is above target stake when their fee bid is lower than the one currently being used and their commitment size changes the LP that meets the target stake: fee changes to fee bid by the LP that is now at the place in the bid order to provide the tarrget stake (<a name="0042-LIQF-026" href="#0042-LIQF-026">0042-LIQF-026</a>)
+- An LP leaves a market that is above target stake when their fee bid is lower than the one currently being used and their commitment size doesn't the LP that meets the target stake: fee doesn't change (<a name="0042-LIQF-027" href="#0042-LIQF-027">0042-LIQF-027</a>)
```
Comment (davidsiska-vega) [inline protocol/0042-LIQF-setting_fees_and_rewarding_lps.md:284]: I am struggling to parse this sentence.

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
Apart from one comment and the linting errors it looks good.

File: protocol/0042-LIQF-setting_fees_and_rewarding_lps.md (hunk 2 above)
Comment (barnabee) [reply protocol/0042-LIQF-setting_fees_and_rewarding_lps.md:284]:
```suggestion
- An LP leaves a market that is above target stake when their fee bid is lower than the one currently being used and their commitment size doesn't change which LP meets the target stake: fee doesn't change (<a name="0042-LIQF-027" href="#0042-LIQF-027">0042-LIQF-027</a>)
```

**Comment by gordsport** [comment]
@vega-paul 
I have fixed the liniting / spelling. Just this review comment to address:

- https://github.com/vegaprotocol/specs/pull/1598#discussion_r1116749182

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**